content/en/monitors/status/graphs.md

```diff
@@ -0,0 +1,98 @@
+
+## Monitor metadata
+
+{{< img src="/monitors/status/graphs/graphs_metadata.png" alt="Monitor metadata section on the right side of the status page." style="width:100%;" >}}
+
+The right panel of the graph section of the monitor status page provides a high level overview of your monitor including
+
+|  | Description |
+| ---- | ---- |
+| Groups  | Count of groups per status (Alert, Warn, No Data, Ok)  |
+| Visualize as | Graph selector to toggle between the Evaluated Data, Source Data, and In Group graphs. |
+| Query | The raw monitor query. A link to the Metrics explorer is also available\*. |
+| Evaluation | The aggregation method applied to the Query with the evaluation window. |
+| Notification count | Count of notifications sent from this monitor. |
+
+[**UPDATE**]
+\* The link to the Metrics Explorer is available only for Metrics, Event, Logs, … monitors.
+
+## Filter the page by groups or status
+
+Depending on the query, the monitor might have multiple groups. To focus on a specific group, use the filter dropdowns to select the desired group.
+
+{{< img src="/monitors/status/graphs/graphs_filter_by_group_status.png" alt="Example monitor status page filtered by the Alert status template variable" style="width:100%;" >}}
+
+You can chose to scope the page on:
+
+Group status
+: Only groups that are currently in the selected state will be shown
+
+Muted state
+: Only groups that are muted, or not, will be shown.
+
+Group tags
+: Only groups that have the selected tag will be shown.
+
+## Evaluated data graph
+
+{{< img src="/monitors/status/graphs/graph_evaluated_data_graph.png" alt="Hover over the line of an evaluated data graph to see the single value." style="width:100%;" >}}
+
+The evaluation graph is specific to the monitor and displays the single values compared to the threshold, aggregated from the query's series of points. For example, if the monitor evaluates the average of the query over the last 5 minutes, each datapoint shows the metric's average value for those 5 minutes. 
+
+It matches your monitor's configuration to accurately show the monitor's status using evaluations settings and the monitor's current status.
+
+## Source data graph
+
+{{< img src="/monitors/status/graphs/graphs_source_data.png" alt="Status page displaying the source data graph" style="width:100%;" >}}
+
+The source graph shows the raw data points submitted to Datadog, used to generate the monitor evaluations. Use this graph to view unaltered data over time and confirm if data fluctuations or anomalies trigger alerts.
+
+Use this graph to identify any discrepancies between raw data and expected metrics, which may indicate data collection or submission issues affecting the monitor's status.
+
+### Restrictions of source data graph
+
+The following monitor types are not supported by the provisional status page:
+
+* \[**TODO**\] Add list of monitor types not supported
```

Comment:
@SimoneT-DD to add list of unsupported monitor types

Reply:
Done!